src/e2e-test/features/source.control.management.sync.apps.feature

```diff
@@ -53,3 +53,35 @@ Feature: Source Control Management - Pulling and pushing applications
     Then Clean up pipeline "test_pipeline2_fll_airport" which is created for testing
     When Open Source Control Management Page
     Then Delete the repo config
+
+  @PIPELINE_EDIT_TEST
+  @SOURCE_CONTROL_MANAGEMENT_TEST
+  Scenario: Edit a simple pipeline and restore a version from git
+    # Setup Repo config
+    When Open Source Control Management Page
+    When Initialize the repository config
+    When Deploy and test pipeline "pipeline_edit_test" with pipeline JSON file "logs_generator.json"
+    # Push version 1 of the pipeline to git.
+    Then Click push button in Actions dropdown
+    Then Commit changes with message "upload pipeline to Git"
+    Then Banner is shown with message "Successfully pushed pipeline pipeline_edit_test"
+    # Edit the pipeline to create a new LCM version.
+    Then Click edit button in Actions dropdown
```

Comment:
nit: We can have one step to create a new version.

Reply:
Done.